backend/models/subscription.models.js

```diff
@@ -9,7 +9,7 @@ const subscriptionSchema = new mongoose.Schema({
   appId: {
     type: Number,
     required: true,
-    ref:'Application'
+    // ref:'Application'
```

Comment:
Any specific reason to remove this reference as it refers as relationship with the applications.

Reply:
when we use ref, the type is supposed to be ObjectId. Since Application Object_Id is number, it is causing error.